scanner/image/db/konflux.Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM registry.redhat.io/rhel8/postgresql-15:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-v4-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-v4-db" \
+    io.openshift.tags="rhacs,scanner-v4-db,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-v4-db-rhel8" \
+    source-location="https://github.com/stackrox/stackrox" \
+    summary="Scanner v4 DB for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+USER root
+
+COPY --chown=postgres:postgres \
+     scanner/image/db/scripts/docker-entrypoint.sh \
+     scanner/image/db/scripts/init-entrypoint.sh \
+     /usr/local/bin/
+
+RUN dnf upgrade -y --nobest && \
+    dnf install -y zstd && \
+    localedef -f UTF-8 -i en_US en_US.UTF-8 && \
+    mkdir -p /var/lib/postgresql && \
+    groupmod -g 70 postgres && \
+    usermod -u 70 postgres -d /var/lib/postgresql && \
+    chown -R postgres:postgres /var/lib/postgresql && \
+    chown -R postgres:postgres /var/run/postgresql && \
```

Comment:
Downstream also does this
```Dockerfile
    # Change ownership of directories that are owned by the original postgres uid.
    # Detect these with `find / -uid 26` in the original container.
    chown -R postgres /var/lib/pgsql && chown -R postgres /opt/app-root && \
```

I wonder if it's needed here too.

Reply:
Indeed. `/var/lib/pgsql` is the former home of the `postgres` user and `/opt/apt-root/src` is the home of the user named `default`.

It looks like Connor first introduced this in downstream central-db in [MR!175](https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/merge_requests/175) and I even reviewed and merged it.

Let's not copy this to Konflux since things work without it.